app/src/main/res/layout-land/topic_lessons_title.xml

```diff
@@ -16,14 +16,10 @@
 
     <TextView
       android:id="@+id/topic_play_text_view"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
+      style="@style/Heading2"
```

Comment:
Doesn't this need to be a view start or center variant? Ditto elsewhere.

(In general, I'd expect all text views to have to pick some view start or view center variant to be explicit about their style).

Reply:
Done.